processing/src/main/java/org/apache/druid/segment/incremental/ParseExceptionHandler.java

```diff
@@ -76,27 +86,51 @@ public void handle(@Nullable ParseException e)
       rowIngestionMeters.incrementProcessedWithError();
     } else {
       rowIngestionMeters.incrementUnparseable();
-    }
+      if (emitter != null) {
+        DateTime timeOfException = new DateTime(e.getTimeOfExceptionMillis());
 
-    logParseExceptionHelper(e);
-
-    if (savedParseExceptionReports != null) {
-      ParseExceptionReport parseExceptionReport = new ParseExceptionReport(
-          e.getInput(),
-          e.isFromPartiallyValidRow() ? "processedWithError" : "unparseable",
-          e.isFromPartiallyValidRow()
-          ? ((UnparseableColumnsParseException) e).getColumnExceptionMessages()
-          : ImmutableList.of(e.getMessage()),
-          e.getTimeOfExceptionMillis()
-      );
-      savedParseExceptionReports.add(parseExceptionReport);
-    }
+        ImmutableMap<String, Object> extraData = new ImmutableMap.Builder().putAll(taskMetadata)
+                                                                           .put("input", e.getInput())
+                                                                           .put("detailMessage", e.getMessage())
+                                                                           .build();
+        Event event = new AlertEvent(
```

Comment:
Should we use a new event type here which is not emitted by default so that existing users who are using emitters do not get bombarded with parsing error alerts suddenly when they update their versions of imply. Or what is protecting against this?

Reply:
this is just getting something there. we will likely make a new event for this

Reply:
Do we want to have a distinction here between "unparseable" and "partially valid" like in the parse exception report below?

```
            e.isFromPartiallyValidRow() ? "processedWithError" : "unparseable",
            e.isFromPartiallyValidRow()
            ? ((UnparseableColumnsParseException) e).getColumnExceptionMessages()
            : ImmutableList.of(e.getMessage()),
```